favourites.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/css/favourites.css" />
+    <title>Travel Companion</title>
```

Comment:
Is that suppose to say Travel Companion?

Reply:
addressed in this commit :) 
2e4127c8ce3a2b66cf01c8c2a90835ddc608e4ad

Reply:
Ok that looks good. Please keep that in mind the next time you do something like this. Possibly read the docs and best practices as well!!

Reply:
But I appreciate the effort :)